projects/TD3D/td3d/td3d.py

```diff
@@ -129,6 +129,14 @@ def loss(self, batch_inputs_dict: dict, batch_data_samples: SampleList,
         annotated_points = []
         for i in range(len(batch_data_samples)):
             gt_pts_seg = batch_data_samples[i].gt_pts_seg
+
+            gt_pts_seg.pts_instance_mask[gt_pts_seg.pts_semantic_mask == 18] = -1 #todo: why 18?
```

Comment:
I am not sure about it. It's very inconveniently to do it in the head, because markup is already combined into a batch there. Maybe we can do it during data processing stage? For example, we can create new augmentation